Add: 1.11.1 release post

LordAro opened the pull request.

TrueBrain commented:
Would you mind adding text similar to #202 for socials? Easy copy/paste makes life easier :)

TrueBrain commented:
![OpenTTD - News #5](https://user-images.githubusercontent.com/1663690/115160438-6848b280-a098-11eb-88ea-79f6716e12f7.png)

Pretty image for Steam. Boring .. I have no inspiration today.

LordAro commented:
Reddit:
```
OpenTTD 1.11.1 just released!

Bugfixes! Also we added volume controls to the Options menu! And added a vsync option!

https://www.openttd.org/news/2021/04/18/openttd-1-11-1.html
```

Discord:
```
Hi @everyone,

OpenTTD 1.11.1 is available now!

Bugfixes! Also we added volume controls to the Options menu! And added a vsync option!

https://www.openttd.org/news/2021/04/18/openttd-1-11-1.html
```

Twitter:
```
OpenTTD 1.11.1 out now!
Bugfixes! And also we added volume controls to the Options menu!
https://www.openttd.org/news/2021/04/18/openttd-1-11-1.html
```

tt-forums:
```
OpenTTD 1.11.0 ready for download!

Bugfixes! Also we added volume controls to the Options menu! And added a vsync option!

https://www.openttd.org/news/2021/04/18/openttd-1-11-1.html
```

Steam:
exactly the same post as on our website with the above image
